Rakefile

```diff
@@ -80,7 +80,8 @@ IMAGES.each do |image|
       info "Running Hadolint to check the style of #{image}/Dockerfile"
       # Ignore the need to pin package versions
       # Ignore use of curl and wget
-      sh "docker run --rm -i lukasmartinelli/hadolint hadolint --ignore DL3008 --ignore DL4000 --ignore DL4001 - < #{image}/Dockerfile"
+      # Ignore the need to pin apk package versions
```

Comment:
I assume these ignore comments correspond to the ignore flags being passed - perhaps it would be useful to be explicit about which value corresponds to which ignore?
e.g. # Ignore the need to pin apk package versions (DL3018)
or whatever

Reply:
Yeah, that's what I've done in https://github.com/puppetlabs/puppet_docker_tools/blob/master/README.md#puppet-docker-lint

I'm not sure how beneficial that is to add in here (there is a super-handy chart at github.com/hadolint/hadolint) since this is being deprecated Real Soon Now, mostly trying to get CI working here so we can get some existing community PRs in.